install.php

```diff
@@ -13,20 +13,23 @@
 use rex_sql;
 use rex_yform_manager_table_api;
 
+use function count;
+
+$sql = rex_sql::factory();
+
 /* Tablesets aktualisieren */
-if (rex_addon::get('yform') && rex_addon::get('yform')->isAvailable()) {
+if (rex_addon::get('yform')->isAvailable()) {
```

Comment:
Geht's hier nicht eh rein um den Manager?

```suggestion
if (rex_plugin::get('yform', 'manager')->isAvailable()) {
```